CMakeLists.txt

```diff
@@ -269,6 +269,8 @@ endif()
 
 if("${SPARSE}" STREQUAL "y")
   list(APPEND TOOLCHAIN_C_FLAGS -D__CHECKER__)
+  # Avoid compiler "attribute directive ignored" warnings
+  list(APPEND TOOLCHAIN_C_FLAGS -Wno-attributes)
```

Comment:
Just noting that the proper way to do this would be through the toolchain abstraction, but since `sparse` is meant to be only used with GCC, there is no point in doing so.